modules/gem5/_posts/2000-03-01-assignment1.md

```diff
@@ -130,7 +121,44 @@ Use `4GHz` as the clock frequency.
 
 In your report, answer the same questions after simulation supported with data. A complete set of simualtion data for this step should include **6 configurations** (2 options for CPU model * 3 options for memory model).
 
-### Step III: General questions
+### Step III: Using different compiler optimizations
+
+In this step, you'll be using different compiler optimizations to run the same
+matrix multiplication program. Compiler optimization flags are options that can
+be used to improve the *performance* of the program at the expense of
+compilation time and the ability to debug the program. The default version of
+the matrix multiply program is compiled with the flag `-O2`:
+
+```sh
+g++ -o mm mm.cpp -static -O2
+```
+
+We have compiled binaries with flags `-O0` and `-O3`. The former reduces the
```

Comment:
`-O0` simply does *no* optimizations. It doesn't "optimize" the compilation time. `-O3` applies most of the optimizations that gcc (or llvm/clang) supports (still not *all* of them, though).

Reply:
fixed it.